packages/EasyErrorHandler/src/Bridge/Symfony/Listener/ExceptionEventListener.php

```diff
@@ -5,29 +5,33 @@
 namespace EonX\EasyErrorHandler\Bridge\Symfony\Listener;
 
 use EonX\EasyErrorHandler\Interfaces\ErrorHandlerInterface;
+use EonX\EasyErrorHandler\Interfaces\Exceptions\TranslatableExceptionInterface;
+use EonX\EasyErrorHandler\Interfaces\TranslatorInterface;
 use Symfony\Component\HttpKernel\Event\ExceptionEvent;
 
 final class ExceptionEventListener
 {
-    /**
-     * @var \EonX\EasyErrorHandler\Interfaces\ErrorHandlerInterface
-     */
-    private $errorHandler;
-
-    public function __construct(ErrorHandlerInterface $errorHandler)
-    {
-        $this->errorHandler = $errorHandler;
+    public function __construct(
+        private ErrorHandlerInterface $errorHandler,
+        private TranslatorInterface $translator
+    ) {
     }
 
     public function __invoke(ExceptionEvent $event): void
     {
-        $this->errorHandler->report($event->getThrowable());
+        $throwable = $event->getThrowable();
+
+        if ($throwable instanceof TranslatableExceptionInterface) {
+            $throwable->setMessage($this->translator->trans($throwable->getMessage(), $throwable->getMessageParams()));
```

Comment:
I think we shouldn't translate the message BEFORE passing a throwable to the error handler. Because in this case, we will try to translate it again in `ExtendedExceptionBuilder`. At least, when we call the `render` method.

If we want the message to be translated when we call the `report` method, then we should do this on lower levels. Also, if we translate only in the Symfony Bridge, your fix won't work for Laravel applications.

@natepage could you please advise a better place to do translation, common for usual logs and Bugsnags? I would say it should be `ErrorDetailsResolver` (in both, `resolveSimpleDetails` and `resolveExtendedDetails` methods, to translate both, `message` and `user_message`). But `LoggerReporter` logs the message "as is" (without passing it through the resolver).

Reply:
We can remove transaction in `ExtendedExceptionBuilder`.

Reply:
@alexndlm @roman-eonx thank you for this PR. The real question is where exactly do we want to see this translated message.

- in logs: is it ok to have it in the context?
- in bugsnag: is it ok to have it in the error tab?
- error 500 page: what is that?

Reply:
@natepage well, I'd say we should see them as "main" messages in both, logs and Bugsnags. But I see that it would be probably harder to implement than with the context or the error tab.

Reply:
I have removed the translation of the exception message from `ExtendedExceptionBuilder`.

Reply:
I still don't like the current solution because:
1. It will work only for Symfony.
2. It introduces a BC because of the changed `TranslatableExceptionInterface` that can be potentially implemented in some of our projects.

@natepage do you have any ideas on where it would be better to translate the message if we want it to work the way described above?

Reply:
@roman-eonx I'm not sure exactly where, but definitely not in the symfony event listener, it has to be part of the core logic of the package and not in a particular bridge. I need to think about it but I've been really busy with AVC lately sorry. I promise I will come back to this shortly.

Reply:
@alexndlm @roman-eonx one more question, what message are you expecting to see there?

- `message`: internal message for devs
- `userMessage`: external message for end users

Reply:
@natepage the first one. The second one is not that important for debugging and can be left untranslated.

Reply:
@alexndlm @roman-eonx I've had a look at this and I have an idea, I will create a separate PR for you to review 😉